lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -2556,3 +2672,119 @@ void mme_app_handle_modify_ue_ambr_request(
   unlock_ue_contexts(ue_context_p);
   OAILOG_FUNC_OUT(LOG_MME_APP);
 }
+
+/**
+ * This Function handles PCRF initiated
```

Comment:
In MME, instead of saying PCRF initiated , say N/W initiated

Reply:
done